src/components/ClearingsUploader/ClearingsUploader.js

```diff
@@ -43,6 +44,28 @@ const propTypes = {
 };
 
 const defaultProps = {
+  messages: {
+    step_1: '1. Elegí la procesadora',
+    step_2: '2. Subí tu archivo',
+    modal: 'Conocé los formatos permitidos',
+    action_text_1: 'Arrastrá acá el archivo',
+    action_text_2: 'o hace click acá y subilo manualmente',
+    link: 'Conocé los formatos permitidos',
+    drag_here: 'Arrastrá acá el archivo',
+    or: 'o hace ',
+    click_here: 'click acá',
+    manual_text: ' y subilo manualmente',
+    uploading: 'Subiendo archivo',
+    upload_success: 'Archivo subido',
+    upload_failure: 'El archivo no subió correctamente',
+    format_error: 'El tipo de archivo no es valido',
+    retry: 'Intentar de nuevo'
+  },
+  options: [
```

Comment:
This default is definitely not ok, we don't want dummy select values in production.

Reply:
done